server/models/articletag.js

```diff
@@ -0,0 +1,20 @@
+export default (sequelize, DataTypes) => {
+  const ArticleTag = sequelize.define('ArticleTag', {
+    articleId: DataTypes.INTEGER,
+    tagId: DataTypes.INTEGER
+  }, {});
+  ArticleTag.associate = (models) => {
+    const { Article, Tag } = models;
+    ArticleTag.belongsTo(Article, {
+      foreignKey: 'articleId',
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE'
```

Comment:
Good work @Darthrighteous. However, I notice you're using `onUpdate: 'CASCADE'`. This is not necessary, in my opinion.

Reply:
I agree, taking it out